framework/build.mk

```diff
@@ -11,6 +11,10 @@ MOOSE_JOBS        ?= 8
 # Include variables defined by MOOSE configure if it's been run
 -include $(MOOSE_DIR)/conf_vars.mk
 
+# PREFIX should be set in conf_vars. If however the user didn't run configure we
+# won't have a prefix. We'll use the automake default then:
+PREFIX ?= '/usr/local'
```

Comment:
I feel like I've been destroyed by defaults like this in the past

Reply:
But I guess this a default for where *you* are installing? Not a default for where to look for packages?

Reply:
`/usr/local` is the normal "automake" default and yes, this is for where you will install, not look for anything. However, since we don't require that users run "configure". We evaluate $PREFIX" in our Makefile and it's empty. So our "no configure" default is actually `/`. If you run "configure" without any arguments, you get an explicit PREFIX of `/usr/local`. To me, it's sane to make those two the same, which this will do.